SortApp/SortApp.Presentation/MainWindow.xaml

```diff
@@ -45,5 +45,12 @@
             <TextBox x:Name="ResultTextBox" Margin="0 5 5 0" IsReadOnly="True" VerticalScrollBarVisibility="Auto" VerticalContentAlignment="Center"
                      TextWrapping="Wrap" Grid.Column="1" Text="{Binding Path=ResultSorting, Mode=OneWay, UpdateSourceTrigger=PropertyChanged}"/>
         </Grid>
+        <DataGrid x:Name="IterationsDataGrid" Grid.Row="5" Margin="5" AutoGenerateColumns="False" IsReadOnly="True" 
+                  ItemsSource="{Binding Iterations}">
+            <DataGrid.Columns >
+                <DataGridTextColumn Header="#" Width="30" Binding="{Binding Number}"/>

```

Comment:
I would suggest to set Width to 'Auto' and set MinWidth="30"
in that way if there are a lot of operations - column will be resized automarically.